Modifiable in game text and in game hints

MattEqualsCoder opened the pull request:
Closes #5

MattEqualsCoder commented:
@Vivelin Okay, I think this is it for me for the v7 changes pretty much. I do want need to add the example game.yml file and add documentation around it. But I think other than it should be good to go.

This adds all of the patches discussed in #228 with the exception of the colorblind doors and the GGG one since they didn't seem to work unfortunately. They'd need more digging into I think to figure out.

I think I'll create a beta here in the next day or two. Do we want betas to be hosted on github and just not announce them in the discord channel? If not, I can host it myself to post in the dev discord.

Vivelin commented on src/Randomizer.Data/Configuration/Yaml/Sassy/game.yml:
```diff
@@ -0,0 +1,147 @@
+
+
+    Dray gone
+- Text: |+
```
This one's duplicate

Vivelin commented on src/Randomizer.Data/Configuration/Yaml/Sassy/game.yml:
```diff
@@ -0,0 +1,147 @@
+    your ISP can see
+    everything you do?
+    Make them Blind
+    to with NordVPN!
```
blind to <ins>it</ins>?

Vivelin commented:
I'm fine with betas being hosted as pre-release… releases here on GitHub.

MattEqualsCoder commented:
Cool. Updated the version number and made these changes, added the game yml file template, and added a bit more messages. I'll try to do a bit more testing tonight or tomorrow morning, then post in the rando dev discord in case in hopes we can get a RC out next Monday.

Vivelin reviewed (dismissed):
The only thing I'm still a little unsure about is using the real names of some of these common sponsors. I'm not feeling very creative but maybe we should change them ever so slightly just so people don't think we're _actually_ sponsored, or something.

Think something along the lines of fake companies you see in stuff but where you still know what they're mimicking.
